panel/src/components/Views/PreviewView.vue

```diff
@@ -0,0 +1,227 @@
+<template>
+	<k-panel class="k-panel-inside k-preview-view">
+		<header class="k-preview-view-header">
+			<k-button-group>
+				<k-button
+					:link="back"
+					:responsive="true"
+					:title="$t('back')"
+					icon="angle-left"
+					size="sm"
+					variant="filled"
+				>
+				</k-button>
+				<k-button icon="title" element="span">
```

Comment:
`Question:` We are using `k-button` here for the styling mainly? (as forcing to span kinda remove the functionality purpose, I'd think)

Reply:
It's the easiest solution to get the icon and I was also thinking that it is a good preparation to turn it into the tree dropdown.